src/fracTrackerMobileLayer.js

```diff
@@ -55,6 +55,10 @@ L.GeoJSON.FracTrackerMobile = L.GeoJSON.extend(
         if (typeof self._map.spin === 'function') {
           self._map.spin(false);
         }
+      }).error(function() {
```

Comment:
@neelesh17, removing the error statement should fix the issue with the failing tests. Could you give it a try? Replacing `error` with `fail` should work as well. You could also add the functionality to stop the spinner after 10seconds to this layer.

Reply:
Okay, I'll give it a try. Thankyou  @crisner.